lib/ansible/module_utils/iosxr.py

```diff
@@ -106,6 +106,11 @@ def load_config(module, commands, commit=False, replace=False, comment=None):
         module.fail_json(msg=err, commands=commands, rc=rc)
 
     rc, diff, err = exec_command(module, 'show commit changes diff')
+    if rc != 0:
```

Comment:
i would assert that if config diff isn't supported just return None and raise an ansible warning if module diff was asked for and the platform doesnt support it